Parsers/News Article Parser.js

```diff
@@ -0,0 +1,37 @@
+/*
+activation_example:!parse_news name of news website
```

Comment:
should this be "name of news website"  or "topic or story to search for"?

Reply:
@chelming This has been addressed in the new commit